08-More_About_ASDF/08-More_About_ASDF.ipynb

```diff
@@ -0,0 +1,415 @@
+{
```

Comment:
<p>To create an ASDF <code>file</code> -&gt; file missing? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/roman-data-workshop/pull/4/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>